apps/api/e2e/tests/user.test.ts

```diff
@@ -2804,33 +2803,33 @@ describe("User API", () => {
       agentClients.push(agentClient);
     }
 
-    // Alpha Agent: (3 big stable trades)
+    // Alpha Agent: (3 small bad trades)
     for (let i = 0; i < 3; i++) {
       await agentClients[0]?.executeTrade({
         fromToken: config.specificChainTokens.eth.usdc,
-        toToken: config.specificChainTokens.eth.eth,
-        amount: "1000",
+        toToken: "0x000000000000000000000000000000000000dead",
```

Comment:
this test just wants deterministic ordering, so we burn tokens in different amounts instead of relying on potentially flaky token behavior